### PR TITLE
bitcoin address generation multisig test

### PR DESCRIPTION
**Pull request by dolgopolovwork**
### Description of the Change
Now we have multisig address generation test

### CODE REVIEW

File: notary-btc-integration-test/src/integration-test/kotlin/integration/btc/BtcMultiAddressGenerationIntegrationTest.kt [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+            integrationHelper.getAccountDetails(
+                "$sessionAccountName@btcSession",
+                environment.btcGenerationConfig.registrationAccount.accountId
+            )
+        val notaryKeys = sessionDetails.entries.filter { entry -> entry.key != ADDRESS_GENERATION_TIME_KEY }
+            .map { entry -> entry.value }
+        val pubKey = notaryKeys.first()
+        Assertions.assertNotNull(pubKey)
+        val wallet = Wallet.loadFromFile(File(environment.btcGenerationConfig.btcWalletFilePath))
+        Assertions.assertTrue(wallet.issuedReceiveKeys.any { ecKey -> ecKey.publicKeyAsHex == pubKey })
```
Comment (Mingela) [inline notary-btc-integration-test/src/integration-test/kotlin/integration/btc/BtcMultiAddressGenerationIntegrationTest.kt]: Why do we use first key? What are the other keys in wallet?

File: notary-commons/src/main/kotlin/sidechain/iroha/util/ModelUtil.kt [hunk 2]
```diff
@@ -69,18 +70,25 @@ object ModelUtil {
      * @param accountId - account to set details
      * @param key - key of detail
      * @param value - value of detail
+     * @param quorum - tx quorum. -1 by default
```
Comment (Mingela) [inline notary-commons/src/main/kotlin/sidechain/iroha/util/ModelUtil.kt]: Why not to set default value 1 and get rid of that `if`

File: notary-btc-integration-test/src/integration-test/kotlin/integration/btc/BtcMultiAddressGenerationIntegrationTest.kt [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+    init {
+        var peerCount = 0
+        integrationHelper.accountHelper.mstRegistrationAccounts.forEach { mstRegistrationAccount ->
+            integrationHelper.addNotary("test_notary_${peerCount++}", "test_notary_address")
+            val environment = BtcAddressGenerationTestEnvironment(
+                integrationHelper,
+                btcGenerationConfig = integrationHelper.configHelper.createBtcAddressGenerationConfig(0),
+                mstRegistrationCredential = mstRegistrationAccount
+            )
+            environments.add(environment)
```
[the remark this answers is not in the document]
Comment (dolgopolovwork) [reply notary-btc-integration-test/src/integration-test/kotlin/integration/btc/BtcMultiAddressGenerationIntegrationTest.kt]: Every service needs its environment. We use environments to run services (see ` environment.btcAddressGenerationInitialization.init()` few lines below) and we also use it to stop services.

File: notary-commons/src/main/kotlin/sidechain/iroha/util/ModelUtil.kt (hunk 2 above)
Comment (dolgopolovwork) [reply notary-commons/src/main/kotlin/sidechain/iroha/util/ModelUtil.kt]: I don't know if it's a good idea or not. Every transaction will contain `setQuorum(1)` then.

File: notary-btc-integration-test/src/integration-test/kotlin/integration/btc/BtcMultiAddressGenerationIntegrationTest.kt (hunk 1 above)
Comment (dolgopolovwork) [reply notary-btc-integration-test/src/integration-test/kotlin/integration/btc/BtcMultiAddressGenerationIntegrationTest.kt]: Oh yeah. You are right. Copy-paste from old(non-multisig) test.

File: notary-commons/src/main/kotlin/sidechain/iroha/util/ModelUtil.kt (hunk 2 above)
Comment (Mingela) [reply notary-commons/src/main/kotlin/sidechain/iroha/util/ModelUtil.kt]: It's clearer and not more complicated than `if` I think

**Pull request merged**